torchmetrics/classification/accuracy.py

```diff
@@ -227,10 +228,10 @@ def update(self, preds: Tensor, target: Tensor) -> None:
         """ returns the mode of the data (binary, multi label, multi class, multi-dim multi class) """
         mode = _mode(preds, target, self.threshold, self.top_k, self.num_classes, self.multiclass)
 
-        if self.mode is None:
+        if not self.mode:
```

Comment:
can this also be some other value that evaluates to `False` (other then `None`)? If not I'd prefer the more explicit check with `is None`

Reply:
False is also not valid option